arctic/store/versioned_item.py

```diff
@@ -1,19 +1,20 @@
 from collections import namedtuple
 
 
-class VersionedItem(namedtuple('VersionedItem', ['symbol', 'library', 'data', 'version', 'metadata'])):
+class VersionedItem(namedtuple('VersionedItem', ['symbol', 'library', 'data', 'version', 'metadata', 'info'])):
```

Comment:
Agree with @jamesblackburn - I'd prefer to avoid pushing this into everything if possible. I guess it is in here because the our graphs just pull the VersionedItem and do everything with that.